src/View/Requirements_Backend.php

```diff
@@ -1421,6 +1440,32 @@ function () use ($fileList, $minify, $type) {
         return $combinedURL;
     }
 
+    /**
+     * Resolves relative paths in CSS files which are lost when combining them
+     *
+     * @param string $content
+     * @param string $filePath
+     * @return string New content with paths resolved
+     */
+    protected function resolveCSSReferences($content, $filePath)
+    {
+        $doResolving = Config::inst()->get(__CLASS__, 'resolve_relative_css_refs');
+        if (!$doResolving) {
+            return $content;
+        }
+        $fileUrl = Injector::inst()->get(ResourceURLGenerator::class)->urlForResource($filePath);
+        $fileUrlDir = dirname($fileUrl);
+        $content = preg_replace_callback('#(url\([\n\r\s\'"]*)([^\s\)\?\'"]+)#i', function ($match) use ($fileUrlDir) {
+            [ $_, $prefix, $relativePath ] = $match;
+            if ($relativePath[0] === '/' || false !== strpos($relativePath, '://')) {
+                return $prefix . $relativePath;
```

Comment:
If this isn't changing anything, can this please just be `return $_` (which would need to be given a nice name like `$fullMatch`) to make it a little clearer we're not actually changing anything? Sorry I didn't pick this up in my first pass.

Reply:
Looks like this is still not changed to `return $fullMatch`?

Reply:
Sorry, I didn't read your comment correctly at first. Fixed that.